### ISSUE TITLE
Delete registry secret code from oadp-operator - OADP 1.2

### ISSUE THREAD

**Issue by weshayutin** [openshift/oadp-operator#921]
For OADP 1.1+ Currently we reconcile secrets for registry environment variables inside oadp-operator repo.

openshift-velero-plugin then from BSL name, refer back to a hardcoded pattern for a secret name that registry retrieves environment variables from.

Removing this reduce complexity in debugging/updating the "plugin registry" component going forward.

**Comment by mateusoliveira43** [comment]
@weshayutin can you please add more info to this issue? where is this code?

**Comment by weshayutin** [comment]
@kaovilai is this something we can add some specifics to?  I think I wrote this up after discussing with you.

**Comment by kaovilai** [comment]
TL;DR check that anything used in https://github.com/openshift/oadp-operator/blob/oadp-1.2/controllers/registry.go is already moved to https://github.com/openshift/openshift-velero-plugin/blob/master/velero-plugins/imagestream/registry.go and eventually remove it from oadp-operator repo, as the recipient of the functions is only the registry.

**Comment by mateusoliveira43** [comment]
and we do not need to check "old" `controllers/registry.go` code, right? looking into the code in master branch should be enough?

**Comment by kaovilai** [comment]
I guess to be safe, only update master then when we cut 1.4 this code disappears from oadp-operator. It's not urgent.

I believe this issue was titled 1.2 when it was not released. No need to cp to 1.2

**Comment by kaovilai** [comment]
Previously registry.go in oadp-operator creates a deployment of https://github.com/migtools/distribution/

Now, registry.go in openshift-velero-plugin creates the same ENV and pass it to [`func NewClient(configString string, envs []string)`](https://github.com/migtools/udistribution/blob/main/pkg/client/client.go#L36) without a separate deployment.
